src/bowling_SC.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+class bowlingGame {
+
+  constructor() {
+    this.rolls = [[], [], [], [], [], [], [], [], [], []];
```

Comment:
Currently each frame is represented by a Javascript Array and we're doing some calculations on each frame. Is there another way we could represent a frame which would allow us to achieve more encapsulation?